Expensify/App
# Fix emoji bold text

### Details
On Google Chrome on Windows render emojis colorless when they have font-weight over 500. 
This update resolves this by wrapping emojis in specific Text components and giving them normal font-weight.

### Fixed Issues

$ [#6772](https://github.com/Expensify/App/issues/6772)

### Tests


1. Open app
2. Check list of reports with unread messages from users that have emojis in their name
4. Verify that their names appear bolded and emojis have color
5. Open specific report and look at messages from the user
6. Verify that users name appear bolded and emojis have color
7. Open users settings 
8. Verify that users name appear bolded and emojis have color



### QA Steps

None


### Tested On


### Screenshots


#### Web
Mac Chrome
<img width="600" alt="web-mac-chrome" src="https://user-images.githubusercontent.com/25391474/148772439-ea6cec14-7850-498d-aa0b-e7fae6033c76.png">
Mac Safari
<img width="600" alt="web-mac-safari" src="https://user-images.githubusercontent.com/25391474/148772479-e3cb959b-3bb3-4e84-9b05-b3e9e30d5895.png">
Win Chrome
<img width="600" alt="web-win-chrome" src="https://user-images.githubusercontent.com/25391474/148772494-df20216a-a2f4-47e3-b94e-7edc95bf9608.png">


#### Mobile Web

##### Android
<img width="300" alt="android-web-allreports" src="https://user-images.githubusercontent.com/25391474/148772799-2ddf264d-152e-4ce5-b3ad-e9864e8b6ade.jpg">
<img width="300" alt="android-web-messages" src="https://user-images.githubusercontent.com/25391474/148772807-2192c80d-fc19-4af1-af1a-54b0d4e688ec.jpg">
<img width="300" alt="android-web-settings" src="https://user-images.githubusercontent.com/25391474/148772816-2fd31ef7-3368-487e-9785-353ef4c830d8.jpg">


##### iOS
<img width="300" alt="ios-web-allreports" src="https://user-images.githubusercontent.com/25391474/148772825-34cab7bf-d184-42ce-a6e1-6f016f9376a6.png">
<img width="300" alt="ios-web-messages" src="https://user-images.githubusercontent.com/25391474/148772840-02d16fd9-c664-48cb-b630-1f4979724a62.png">
<img width="300" alt="ios-web-settings" src="https://user-images.githubusercontent.com/25391474/148772848-5674ab38-df01-4b20-8315-b4a781a29ee9.png">


#### Desktop
<img width="750" alt="desktop" src="https://user-images.githubusercontent.com/25391474/148773194-8ecfbb47-c77c-4826-8861-d24de793a316.png">


#### iOS
<img width="300" alt="native-ios-allreports" src="https://user-images.githubusercontent.com/25391474/148773134-781193a9-6abf-46b0-89f6-920cfe26f0a6.png">
<img width="300" alt="native-ios-messages" src="https://user-images.githubusercontent.com/25391474/148773141-16d21234-dbcc-46a7-b183-695b3a793680.png">
<img width="300" alt="native-ios-settings" src="https://user-images.githubusercontent.com/25391474/148773146-25632be4-7492-4273-8d31-07eb8e578684.png">


#### Android
<img width="300" alt="native-android-allreports" src="https://user-images.githubusercontent.com/25391474/148773113-b6d2a094-351c-4975-83d4-bf87ae44a73d.png">
<img width="300" alt="native-android-messages" src="https://user-images.githubusercontent.com/25391474/148773119-5d0a024d-27ed-48f5-ba49-7b16124263ce.png">
<img width="300" alt="native-android-settings" src="https://user-images.githubusercontent.com/25391474/148773125-2b13f4f0-a7ab-4334-b9ae-fecb46701ae6.png">
